src/components/SideNavigation/SideNavigation.css

```diff
@@ -0,0 +1,58 @@
+.side-navigation-container{
+    max-width:12rem;
+}
+
+.side-nav-link{
+    color:var(--primary-color);
+    font-size: 1.4rem;
+    border-radius: var(--border-rd-sm);
+    width: fit-content;
+    margin: 1.2rem 0;
+    padding:0.2rem 0.8rem;
```

Comment:
px for less than 1rem

Reply:
Yeah just for consistency kept it in this manner